packages/katello/katello-installer-base/katello-installer-base.spec

```diff
@@ -2,7 +2,7 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-# %%global prerelease .rc1
```

Comment:
It seems wrong we don't set this in develop. Probably something to look into. Perhaps we also copied in some released version